Fourth set of Scrimba links (#42342)
mdn/content: files/en-us/learn_web_development/core/css_layout/grids/index.md

{{EmbedLiveSample('simple-grid_2', '100%', "130") }}

## Interactive grid lesson

> The following embedded content from Scrimba <sup>[_MDN learning partner_](/en-US/docs/MDN/Writing_guidelines/Learning_content#partner_links_and_embeds)</sup> provides an interactive lesson summarizing the basics of CSS Grid, which also allows you to play with a live grid example and familiarize yourself with how the code works.

Reviewer: - Shall we also add a link on Scrimba?
- Added a suggestion to split the long sentence
```suggestion
The following embedded content from [Scrimba](https://scrimba.com/learn/frontend?via=mdn) <sup>[_MDN learning partner_](/en-US/docs/MDN/Writing_guidelines/Learning_content#partner_links_and_embeds)</sup> provides an interactive lesson on the basics of CSS grid. It also includes a live grid example that you can play with and see how the code works.
```
